cnxarchive/tests/data/data.sql

```diff
@@ -557,6 +572,7 @@ INSERT INTO module_files VALUES (4, 23, 'PhET_Icon.png');
 INSERT INTO module_files VALUES (6, 23, 'PhET_Icon.png');
 INSERT INTO module_files VALUES (9, 23, 'PhET_Icon.png');
 INSERT INTO module_files VALUES (10, 23, 'PhET_Icon.png');
+INSERT INTO module_files VALUES (20, 109, 'index.cnxml.html');
```

Comment:
Another way to look at this is... At this time `modules` records that are of type `portal_type = 'Module'` will not have a `module_files` entry for the collated content. That's why I'm questioning whether this CompositeModule entry should exist.

If we were to add the collated content entries to `module_files`, it should maybe have a new filename?